arches/app/datatypes/url.py

```diff
@@ -267,3 +264,10 @@ def default_es_mapping(self):
                 },
             }
         }
+
+    def pre_tile_save(self, tile, nodeid):
+        try:
+            if "url_label" not in tile.data[nodeid]:
+                tile.data[nodeid]["url_label"] = ""
+        except AttributeError:
```

Comment:
Is the AttributeError exception because `tile` could be None? If so, I think this would be clearer:
```
if tile and "url_label" not in tile.data[nodeid]:
```

Reply:
Fixed in https://github.com/archesproject/arches/pull/10642/commits/aa9e9d9f672af6de7c6924c2e80ab86541676adf